MapboxNavigation.podspec

```diff
@@ -45,7 +45,7 @@ Pod::Spec.new do |s|
   s.dependency "MapboxCoreNavigation", "#{s.version.to_s}"
   s.dependency "MapboxDirections.swift", "~> 0.9.0"
   s.dependency "Mapbox-iOS-SDK", "~> 3.5"
-  s.dependency "OSRMTextInstructions", "~> 0.1.1"
+  s.dependency "OSRMTextInstructions", "~> 0.1"
```

Comment:
The use of a patch version here was intentional: #150. v0.2 isn’t guaranteed to be backwards-compatible with v0.1.x.